pkgs/servers/web-apps/freshrss/extensions/default.nix

```
@@ -0,0 +1,137 @@
+{ config
+, lib
+, fetchFromGitHub
+, fetchFromGitLab
+, freshrss-utils
```

And then add this dependency here instead
```suggestion
, freshrss-utils
, callPackage
```